main/src/ca/uwaterloo/flix/language/phase/Lowering.scala

```diff
@@ -540,19 +547,22 @@ object Lowering {
 
     case Expression.NewChannel(exp, tpe, pur, eff, loc) =>
       val e = visitExp(exp)
-      val t = visitType(tpe)
-      Expression.NewChannel(e, t, pur, eff, loc)
+      val unboxedChannelType = Type.mkEnum(Enums.ChannelMpmc, e.tpe :: Nil, loc)
+      val newChannel = Expression.Def(Defs.ChannelNew, Type.mkImpureArrow(Type.Int32, unboxedChannelType, loc), loc)
+      Expression.Apply(newChannel, e :: Nil, unboxedChannelType, pur, eff, loc)
 
     case Expression.GetChannel(exp, tpe, pur, eff, loc) =>
       val e = visitExp(exp)
-      val t = visitType(tpe)
-      Expression.GetChannel(e, t, pur, eff, loc)
+      // val unboxedChannelType = Type.mkEnum(Enums.ChannelMpmc, e.tpe :: Nil, loc)
+      val getChannel = Expression.Def(Defs.ChannelGet, Type.mkImpureArrow(Types.ChannelMpmc, Types.Boxed, loc), loc)
```

Comment:
Channel doesn't use Boxed more, you can see [here](https://github.com/flix/flix/blob/master/main/src/library/Concurrent/Channel.flix#L177) that Channel just returns its element type outright